Services/gRPC/cntk-image-recon/README.md

```diff
@@ -36,20 +36,44 @@ $ cd ../Services/gRPC/cntk-image-recon
 
 ### Running the service:
 
-To get the `YOUR_AGENT_ADDRESS` you must have already published a service (check this [link](https://github.com/singnet/wiki/tree/master/tutorials/howToPublishService)).
+To get the `ORGANIZATION_NAME` and `SERVICE_NAME` you must have already published a service (check this [link](https://github.com/singnet/wiki/tree/master/tutorials/howToPublishService)).
+
+Create the `SNET Daemon`'s config JSON file (`snetd.config.json`).
 
-Create the `SNET Daemon`'s config JSON file. It must looks like this:
 ```
-$ cat snetd_image_recon_service_config.json
 {
    "PRIVATE_KEY": "1000000000000000000000000000000000000000000000000000000000000000",
-   "DAEMON_LISTENING_PORT": 7004,
+   "DAEMON_LISTENING_PORT": DAEMON_PORT,
+   "DAEMON_END_POINT": "DAEMON_IP:DAEMON_PORT",
    "ETHEREUM_JSON_RPC_ENDPOINT": "https://kovan.infura.io",
-   "PASSTHROUGH_ENABLED": true,
-   "PASSTHROUGH_ENDPOINT": "http://localhost:7003",
+   "IPFS_END_POINT": "http://ipfs.singularitynet.io:80",
    "REGISTRY_ADDRESS_KEY": "0x2e4b2f2b72402b9b2d6a7851e37c856c329afe38",
+   "PASSTHROUGH_ENABLED": true,
+   "PASSTHROUGH_ENDPOINT": "SERVICE_GRPC_IP:SERVICE_GRPC_PORT",  
```

Comment:
SERVICE_GRPC_IP can be a misleading name.

Reply:
SERVICE_GRPC_HOST?

Reply:
Ok